package.json

```diff
@@ -23,23 +23,21 @@
   "bugs": {
     "url": "https://github.com/soywod/pin-field/issues"
   },
-  "main": "lib/pin-field.min.js",
-  "types": "lib/pin-field.d.ts",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
   "files": [
-    "lib/pin-field.min.js",
-    "lib/pin-field.d.ts"
+    "dist",
+    "src"
```

Comment:
Is this line necessary? I means that all the sources are packed with releases, which increases the size of the lib. Maybe `dist` is enough. What do you think?